src/test/feature/Ranger/test_ranger.cpp

```diff
@@ -24,19 +24,34 @@
 
 #include "lib/command.h"
 #include "lib/gpfdist.h"
-#include "lib/sql_util.h"
 #include "lib/string_util.h"
 
 using std::vector;
 using std::string;
 using hawq::test::SQLUtility;
 using hawq::test::Command;
 
+TestHawqRanger::TestHawqRanger()
+{
+	initfile = hawq::test::stringFormat("Ranger/sql/init_file");
+	rangerHost = getRangerHost();
+}
+
+std::string& TestHawqRanger::getRangerHost()
```

Comment:
The RPS and HAWQ master are required to installed on a same node. This test case is to add/remove policy from Ranger Admin, but HAWQ only knows and interacts with RPS, it need not configure Ranger Admin address/hostname. 

I am considering still use a environment value to control it.